helium/src/main/groovy/com/stanfy/helium/dsl/TypeDsl.groovy

```diff
@@ -41,6 +41,9 @@ class TypeDsl {
     if (args.containsKey("skipUnknownFields")) {
       msg.skipUnknownFields = args.skipUnknownFields
     }
+    if (args.containsKey("parent")) {
```

Comment:
```parent``` is more explanatory, evocative and commonly used, and code generated by helium has inheritance anyway, so I think applying 'parent' as concept is more appropriate.